controllers/reps.js

```diff
@@ -0,0 +1,9 @@
+const log = require('../models/reps');
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('reps/index')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  res.send("reps/index")
```